packages/backend/src/payment-method/ilp/connector/core/middleware/max-packet-amount.ts

```diff
@@ -14,7 +14,7 @@ export function createIncomingMaxPacketAmountMiddleware(): ILPMiddleware {
     if (maxPacketAmount) {
       const amount = request.prepare.intAmount
       if (amount > maxPacketAmount) {
-        logger.warn(
+        logger.error(
```

Comment:
I was back and forth on this one, but I _think_ error is correct here, since it would probably require actioning to update the configuration for the peer